keps/sig-auth/20190730-oidc-discovery.md

```diff
@@ -195,10 +243,70 @@ configuration:
 ```
 
 The API server would treat these as `nonResourceURLs`, and restrict access
-appropriately. We will consider expanding `system:public-info-viewer` RBAC
-ClusterRole to grant access to the new paths; some other roles may have
-permission already via a pattern match on `nonResourceURLs` (e.g.
-`cluster-admin`).
+appropriately. We will provide a default RBAC `ClusterRole` called
+`service-account-issuer-discovery` that provides `GET` access to these
+`nonResourceURLs`, but will *not* provide a default `ClusterRoleBinding`. This
+leaves the decision of who is allowed to access these endpoints up to cluster
+admins. A default binding requires further discussion, including ongoing efforts
+to harden the unauthenticated API surface area.
+
+### Risks and Mitigations
+
+- Security is being reviewed by @mikedanese and @liggitt.
+- This feature exposes public key information that is derived from sensitive
```

Comment:
Can we have a test to confirm no private keys are emitted via the JWKS URI?

Reply:
Yes, that's a good idea. Added to 0->alpha graduation criteria.